client/my-sites/themes/controller-logged-in.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+import { startsWith } from 'lodash';
```

Comment:
What a lonely `lodash` instance here. It should be pretty straightforward to get rid of it now that we're creating this file?

Reply:
> What a lonely `lodash` instance here.

Yes, would be a shame if anything happened to it! 🙂 I removed all Lodash (also `compact` and `isEmpty`) from theme controllers in 075a709.